ui/src/components/banner/BannerContainer.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from 'react';
+
+import Banner from './Banner';
+
+const BannerContainer = () => {
+  const [banners, setBanners] = useState([]);
+
+  const bannerData = process.env.REACT_APP_BANNERS;
+
+  useEffect(() => {
+    if (bannerData) {
+      try {
+        const parsedBanners = JSON.parse(bannerData);
+        setBanners(parsedBanners);
+      } catch (e) {
+        console.error('Failed to parse REACT_APP_BANNERS env var.');
+      }
+    }
+  }, [bannerData]);
+
+  return (
+    <>
```

Comment:
@mistryrn may I suggest using a semantically grouping tag here? 

you could have a `section` full of `article`s (banners)